service/src/main/java/bio/terra/cbas/config/CbasInitializationConfiguration.java

```diff
@@ -0,0 +1,18 @@
+package bio.terra.cbas.config;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ConfigurationProperties(prefix = "cbas.initialization")
+public class CbasInitializationConfiguration {
+  private Boolean enabled;
```

Comment:
I added it so that tests could disable it if necessary. it seemed like the two best options were either to set a dummy workspaceCreatedDate or use this "enabled" flag, and setting a dummy workspaceCreatedDate in a test that didn't touch that value (like TestSamService) seemed strange to me. But let me know if you disagree - I was pretty torn on it.